online-shop/src/app/classes.ts

```diff
@@ -0,0 +1,18 @@
+export interface Product{
+id:String;
```

Comment:
I agree with the first 2. For the last one, I'd actually need at least 4 interfaces, because some of the back-end methods have slightly different attribute names, while others also require the quantity attribute. Having so many classes with similar names seems confusing rather than clear to me.